### PR TITLE
Update psammead readme

### LINKED ISSUE #141: Documentation - Component usage example
**Is your feature request related to a problem? Please describe.**
We should have an example of using components

**Describe the solution you'd like**
- Add a `## Components` section to the readme (or below the Components section in #137, once it's been done)
- Give an example of how to combine at least two Psammead components.
- For testing, share it with a dev in another team to check it's understandable. 

**Testing notes**
[Tester to complete]

Dev insight: Documentation needs to make sense to a new joiner. 
- For testing, share the PR with a dev in another team to check it's understandable. 

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by thekp**
Resolves #141

**Overall change:** _Refactoring our Psammead readme doc to be less cluttered on the main readme_

**Code changes:**

- _move info about psammead components to a dedicated readme for it_

- _i've removed the following section https://github.com/bbc/psammead/blob/latest/README.md#nut_and_bolt-using-psammead-components because it looks like we already have the babel plugin installed and setup [here](https://github.com/bbc/psammead/blob/latest/babel.config.js#L26L34)_

- _add info on how to use psammead components in another project_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/README.md [hunk 1]
```diff
@@ -1,21 +1,96 @@
...
+
+[See documentation on the Styled Components site](https://www.styled-components.com/docs/tooling#babel-plugin)
+
+**NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups such as Psammead's. Failing that, make sure your application's `styled-components` dependency is the same version as that in Psammead.
+
+## Developing with Psammead
+
+1. Find a Psammead component you wish you to use, in your project from the [list of available packages](https://github.com/bbc/psammead/tree/latest/packages/components).
```
Comment (dr3) [inline packages/components/README.md]:
```suggestion
1. Find a Psammead component you wish you to use, in your project from the [directory of available packages](https://github.com/bbc/psammead/tree/latest/packages), or by running `npx lerna list` inside the repo.
```

**Review by dr3 — APPROVED** [review]

**Comment by amywalkerdev** [comment]
If we're updating the psammead readme can we update the contact section?

> Psammead is currently maintained by developers in the BBC Articles and Reach + Languages teams. If you want to open an issue, please add it to our issues page. Contact us by email on PsammeadMaintainers@bbc.co.uk, or find us on Slack at #psammead in the bbcnews workspace.

Should probably be `maintained by developers in the BBC Simorgh teams` (or News Web?)  and `or find us on Slack at #si_repo-psammead`

**Review by amywalkerdev — APPROVED** [review]
👍

**Comment by amywalkerdev** [comment]
No testing needed, its a readme change

**Pull request merged**